client/src/store/modules/i18n/en.js

```diff
@@ -42,5 +42,7 @@ export default {
     FIELD_IS_REQUIRED: 'field is required.',
     FIELD_MUST_BE_MORE_THAN_VALUE: ' must be more than value characters',
     FIELD_MUST_BE_LESS_THAN_VALUE: ' must be less than value characters',
-    WRONG_EMAIL_FORMAT: 'Wrong Email format'
+    WRONG_EMAIL_FORMAT: 'Wrong Email format',
+    SITE_NAME: 'Shedulia',
```

Comment:
no, your localization constant must reflect the content, so not SITE_NAME, but SCHEDULIA